### PR TITLE
update(ShareDID): Let share did option as not danger

### LINKED ISSUE #1537: bug(QuickProfile): SharedDID should not be highlighted red
**Issue:**
Within the quickProfile menu when you hover over ShareDID its highlighted red but when you hover over Remove Friend or Block its highlighted normal color 

Maybe instead we can separate the Remove Friend/Block Friend since they are more "Danger" options and when you hover over them they should be highlighted red 

**Screen recording:**

https://github.com/Satellite-im/Uplink/assets/93608357/b104e7df-c17d-4acb-b95c-971f5eb07b84

### PR DESCRIPTION
**Pull request by lgmarchi**
### 1. Let share did option as not danger

https://github.com/Satellite-im/Uplink/assets/63157656/c10b7efc-5b73-45cd-a890-15af6bb0bfe0

### Which issue(s) this PR fixes 🔨

- Resolve #1537

### CODE REVIEW

File: kit/src/components/context_menu/mod.rs [hunk 1]
```diff
@@ -42,7 +42,9 @@ pub fn ContextItem<'a>(cx: Scope<'a, ItemProps<'a>>) -> Element<'a> {
         return None;
     }
 
-    let class = if cx.props.danger.is_some() {
+    let danger_option = cx.props.danger.unwrap_or_default();
+
+    let class = if danger_option {
```
Comment (sdwoodbury) [inline kit/src/components/context_menu/mod.rs]: this could be a 1 liner

**Review by sdwoodbury — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Comment by stavares843** [comment]
neat 🎉

File: kit/src/components/context_menu/mod.rs (hunk 1 above)
Comment (lgmarchi) [reply kit/src/components/context_menu/mod.rs]: Hehe it is true.

**Review by stavares843 — COMMENTED** [review]
missing linter 🔨 

Captura de ecrã 2023-11-22, às 21 27 37

**Review by dariusc93 — APPROVED** [review]

File: ui/src/layouts/chats/presentation/quick_profile/mod.rs [hunk 2]
```diff
@@ -455,17 +468,7 @@ pub fn QuickProfileContext<'a>(cx: Scope<'a, QuickProfileProps<'a>>) -> Element<
...
-                            danger: true,
-                            icon: Icon::Link,
-                            text: get_local_text("friends.share"),
-                            aria_label: "friends-share".into(),
-                            onpress: move |_| {
-                                share_did.set(Some(did_cloned_2.clone()));
-                            }
-                        })
-                    },
+
```
Comment (dariusc93) [inline ui/src/layouts/chats/presentation/quick_profile/mod.rs]:
```suggestion
```

**Pull request merged**